colour/models/rgb/transfer_functions/log.py

```diff
@@ -1,9 +1,9 @@
 #
 """
-Log2 Shaper Implementation
+Common Log Encodings
 ==========================
```

Comment:
Careful at title marker length: `==========================` should be shorter

Reply:
Ah, should have taken care. Noted anyways!